ONSdigital/takeon-business-layer
# Added Dateadjustment to snapshot

# Description
Changes are made to the Business Layer to provide Date Adjustment data to the snapshot
# How to test?
Environment : spp828
Lambda: spp-es-takeon-egestion-spp828-export
Place the following input in the input queue of snapshot 'spp-es-takeon-egestion-spp828-db-export-input'
{ "snapshot_id": "14e0fb27-d450-44d4-8452-9f6996b00e27",
"surveyperiods": [
{ "survey": "023", "period": "201904" },
{ "survey": "023", "period": "201903" },
{ "survey": "066", "period": "201903" },
{ "survey": "066", "period": "201906" }
] }
Verify the success or failure of the snapshot in the output queue '.spp-es-takeon-egestion-spp828-db-export-output'
And also verify the S3 bucket. You can see a snapshot file if the process is success
spp-es-takeon-egestion-spp828-export

# Links
https://collaborate2.ons.gov.uk/jira/browse/SPP-928
